src/components/services-view/services-view.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ServicesViewController implements angular.IController {
     }
 
     this.ctrl.currentFilter = category;
-    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || 'all');
+    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || null);
```

Comment:
Hi, what is the desired behavior here?  

- Clicking on 'All' or 'Other' shows those items, no sub-categories
- Clicking on a main Category shows sub-categories, but no items (unless there is only one sub-category, then show the items)
- Clicking on a sub-category toggles the display of it's catalog items